ext/AvizExt/viz/viz.jl

```diff
@@ -11,11 +11,19 @@ using GLMakie.Colors
 
 Extract time step labels, ensuring last entry is always included.
 """
-function _time_labels(labels)
-    tick_pos = vcat(1:5:length(labels), [length(labels)])
-    tick_label = vcat(string.(labels)[1:5:end], string.(labels)[end])
+function _time_labels(labels; label_step=5)
+    labels_length = length(labels)
+    labels_strings = string.(labels)
 
-    return tick_pos, tick_label
+    tick_position = collect(1:label_step:labels_length)
+    tick_label = collect(labels_strings[1:label_step:end])
```

Comment:
Are these `collect()` necessary?

Reply:
It depends. If I remove them the code still works, but the `typeof` these `1:label_step:labels_length` is `StepRange{Int64, Int64}` and that means that this function would return different types of objects depending on the `(labels_length - 1) % label_step != 0` being `true` or `false`.

So, in my opinion, we should keep these.